core/templates/dev/head/editor/feedback_tab/FeedbackTab.js

```diff
@@ -96,45 +114,76 @@ oppia.controller('FeedbackTab', [
       backdrop: true,
       size: 'lg',
       resolve: {
-        suggestion: function() {
-          return $scope.activeThread.suggestion;
+        suggestionInfo: function() {
```

Comment:
Suggest breaking this into two separate things -- it's not so good to pass big dict blobs if possible.